rules/windows/registry/registry_set/registry_set_persistence_appx_debugger.yml

```diff
@@ -0,0 +1,26 @@
+title: Windows Registry Persistence DebugPath
+id: df4dc653-1029-47ba-8231-3c44238cc0ae
+status: experimental
+description: Detects potential persistense using Appx DebugPath
+references:
+    - https://oddvar.moe/2018/09/06/persistence-using-universal-windows-platform-apps-appx/
+    - https://github.com/rootm0s/WinPwnage
+author: frack113
+date: 2022/07/27
+logsource:
+    category: registry_set
+    product: windows
+detection:
+    selection: 
+        EventType: SetValue
+        TargetObject|contains: 'Classes\ActivatableClasses\Package\Microsoft.'
```

Comment:
Maybe we could also add the "PackagedAppXDebug" technique described in the blog?

Example in the "Bonus Discovery" section 

`reg add HKCU\Software\Microsoft\Windows\CurrentVersion\PackagedAppXDebug\Microsoft.People_10.1807.2131.0_x64__8wekyb3d8bbwe /d "C:\windows\system32\cmd.exe"
`

Reply:
I mixed up the path a little bit